rundeckapp/grails-app/assets/javascripts/resourceModelConfig.js

```diff
@@ -231,7 +231,7 @@ addConfig: function(type) {
 checkForm: function () {
     if ($('configs').down('button.needsSave')) {
         var p=[];
-        $('configs').select('button.needsSave').each(function(e) {
+        $('configs').on('select','button.needsSave').each(function(e) {
```

Comment:
I think the prototype code `$(content).select('input')` should just be converted to `jQuery('input')`